### PR TITLE
Doc: notes about retry logic

### PR DESCRIPTION
**Pull request by j0tunn**

### CODE REVIEW

**Comment by j0tunn** [comment]
@SwinX @sipayRT

File: doc/config.md [hunk 1]
```diff
@@ -191,6 +191,14 @@ Settings list:
 * `retry` – maximum amount of relaunch fallen tests with a critical error. If not
   specified, the fallen tests will not be relaunched (by default it's 0).
 
+  Note that fallen test will not be retried immediatelly, it will be delayed until
+  all tests finish and then restarted in new browser session. So the tests will be
+  performed in waves:
+  - all tests launch
+  - first retry launch of fallen tests
+  - second retry launch of tests fallen in first retry launch
```
Comment (sipayRT) [inline doc/config.md]: in first retry launch --> in previous launch

File: doc/config.md [hunk 2]
```diff
@@ -191,6 +191,14 @@ Settings list:
 * `retry` – maximum amount of relaunch fallen tests with a critical error. If not
   specified, the fallen tests will not be relaunched (by default it's 0).
 
+  Note that fallen test will not be retried immediatelly, it will be delayed until
+  all tests finish and then restarted in new browser session. So the tests will be
+  performed in waves:
+  - all tests launch
+  - first retry launch of fallen tests
```
Comment (SwinX) [inline doc/config.md]: `run failed tests using new browser sessions first time`

Comment (SwinX) [inline doc/config.md]: можно такой вариант:

```
run tests failed during previous test run
run tests failed during first retry
...
```

**Comment by j0tunn** [comment]
:up:

**Comment by SwinX** [comment]
:ok:

**Comment by sipayRT** [comment]
:ok:

**Pull request merged**